modules/govuk_jenkins/manifests/init.pp

```diff
@@ -36,7 +36,7 @@
   $plugins = {},
   $ssh_private_key = undef,
   $ssh_public_key = undef,
-  $version = '1.554.2',
+  $version = undef,
```

Comment:
What's the behaviour if we don't specify a version? is it better to pin here rather than having another line in hiera?